QModManager/API/SMLHelper/Utility/StorageHelperExtensions.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace QModManager.API.SMLHelper.Utility
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Extension methods to provide static calls for <see cref="ItemsContainer"/> into <see cref="ItemStorageHelper"/> methods.
+    /// </summary>
+    public static class StorageHelperExtensions
```

Comment:
This storage helper's all over the place, it's in Interfaces, it's in Utility, and it's in Utility again. Why so many files?

Reply:
@DingoDjango Did a lot of these so I'm not all that sure.  
The Extensions look to mostly be there for convenience.

Reply:
I put everything in the same folder as far as I remember. We had an interface as PS requested, a helper class which did the actual work and a static class which exposed methods to intellisense.